src/components/articles/article/ArticleFrame.js

```diff
@@ -15,67 +16,82 @@ const ArticleFrame = (props) => {
-        setScreenCoverOn(true);
-    });
+        navigator.clipboard.writeText(window.location.href).then(() => {
+            setScreenCoverOn(true);
+        });
+    }
+
+    function moveToComments(){
+        window.scrollTo(0, document.body.scrollHeight);  
+     setShowComments(true);
     }
 
     function clossScreen() {
         setScreenCoverOn(false);
     }
 
     return (
-    <>
+      <>
         {screenCoverOn && (
-            <ScreenCover
+          <ScreenCover
             declineFunc={null}
             confirmFunc={clossScreen}
             declineText={null}
             confirmText="OK"
             title="URL Copied !"
             content={null}
             _class={null}
-            />
+          />
         )}
 
         <div className="article_frame">
-            <p className="article_topic">{topic}</p>
-            <h1 className="article_title">{title}</h1>
+          <p className="article_topic">{topic}</p>
+          <h1 className="article_title">{title}</h1>
+
+          <p className="article_author">
+            <span className="material-symbols-outlined">person</span>
+            {author}
+          </p>
+          <p className="article_create_date">{created_at}</p>
 
-            <p className="article_author">
-                <span className="material-symbols-outlined">person</span>
-                {author}
+          <div className="button_group">
+            <p className="article_comment" onClick={moveToComments}>
+              <span className="material-symbols-outlined">comment</span>
+              comments({comment_count})
             </p>
-            <p className="article_create_date">{created_at}</p>
-
-            <div className="button_group">
-                <p className="article_comment">
-                    <span className="material-symbols-outlined">comment</span>
-                    comments({comment_count})
-                </p>
-
-                <p className="article_votes">
-                    <span className="material-symbols-outlined">thumb_up</span>
-                    votes({votes})
-                </p>
-                <p className="article_share" onClick={copyToClipboard}>
-                    <span className="material-symbols-outlined">link</span>
-                    share
-                </p>
-            </div> 
-            
-            <RandomImage _class="article_img" />
-
-            <p className="article_img_description">
-                (Image is randomly picked From BBC News website)
+
+            <p className="article_votes">
+              <span className="material-symbols-outlined">thumb_up</span>
+              votes({votes})
+            </p>
+            <p className="article_share" onClick={copyToClipboard}>
+              <span className="material-symbols-outlined">link</span>
+              share
             </p>
+          </div>
+
+          <RandomImage _class="article_img" />
```

Comment:
When you copy the link this component is re-rendered and picks a new random image. Consider having it only pick a new image when the `article_id` changes so it stays the same on each article. Otherwise I like the idea of it.